exporter/collector/testdata/fixtures/workload_metrics_expect.json

```diff
@@ -6307,11 +6326,7 @@
               }
             }
           ]
-        }
-      ]
-    },
-    {
-      "timeSeries": [
```

Comment:
This looks problematic, the OC exporter was splitting this into two request batches while we are just sending one..

Reply:
I thought I was using the batch processor when I generated these fixtures?  Is the batch processor not splitting correctly?

Reply:
Discussed with David. What's happening is the batchprocesor gives 200 OTLP points, but we split the summary points into multiple timeseries, which gives us more than the 200 metric limit. Possible fixes are:

1. Implement batching in the exporter, which is how the OC exporter got around this, or
2. Unpack the summary points into multiple timeseries in a new processor *before* batchprocessor

Reply:
1. also gives us the opportunity to deal with other oddities around sending metric data points (e.g. multiple points in the same timeseries).  I think this has my vote as an inevitability here.   Given we can't really control how users set up processors, I think we should aim for this exporter to be as resilient as possible to things users will shove at it.

Reply:
Just to see where it went, I opened https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7134.  But implementing batching in the exporter (could we use the batch processor as a library?) is definitely the most expedient solution.

Reply:
Discussed at standup. We are implement some batching in our exporter for expediency and maybe revisit this in the future depending what happens with David's issue. Not clear yet if we want to try and re-use the batchprocessor's batching code or do something bespoke based on GCM's unique batching requirements.

We will probably stop recommending users use `batchprocessor` in their pipeline though.